### ISSUE TITLE
export non-editor related content

### ISSUE THREAD

**Issue by bartart3d** [olivierkes/manuskript#364]
Hi all, 

Another feature request but perhaps I'm missing something, I seem not to be able to export things like the summary, content from the character development, plots, world items etc.. , only content that is set to compile in the Editor. I'm sure it could come in handy at some point to export all content to another format like docx, pdf, etc.. for collaboration purposes, or when writing scripts for video, theater etc... 

I can copy paste the text in another editor of course, but that feels quite counterproductive.

grtz,
Bart

**Comment by TiggsPanther** [comment]
Something like this would definitely be helpful. Being able to export a _Story Bible_ of sorts.

**Comment by zmike** [comment]
I agree. It would be great to be able to export stuff out of notes and detailed info for characters.

**Comment by TheJackiMonster** [comment]
This seems to be a duplicate of #330.
